src/components/interfaces/HMI_API.xml

```diff
@@ -5837,4 +6276,122 @@
 </function>
 
 </interface>
+
+<interface name="AppService" version="1.0.0" date="2019-01-20">
+    <description>Interface used for interacting with app services as a producer or consumer</description>
+    <function name="PublishAppService" messagetype="request">
+        <description>Registers a service offered by this app on the module</description>
+
+        <param name="appServiceManifest" type="Common.AppServiceManifest" mandatory="true">
+            <description> The manifest of the service that wishes to be published.</description>
+        </param>
+    </function>
+
+    <function name="PublishAppService" messagetype="response">
+        <description>Response to the request to register a service offered by this app on the module</description>
+
+        <param name="appServiceRecord" type="Common.AppServiceRecord" mandatory="false">
+            <description> If the request was successful, this object will be the current status of the service record for the published service. This will include the Core supplied service ID.</description>
+        </param>
+    </function>
+
+    <function name="OnAppServiceData" messagetype="notification">
+        <description>This notification includes the data that is updated from the specific service. HMI->SDL if the HMI is a producer, SDL->HMI if the HMI is a consumer</description>
+
+        <param name="serviceData" type="Common.AppServiceData" mandatory="true"/>
+    </function>
+
+    <function name="GetAppServiceData" messagetype="request">
+        <description> This request asks the module for current data related to the specific service. It also includes an option to subscribe to that service for future updates</description>
+        
+        <param name="serviceType" type="String" mandatory="true"/>
+        
+        <param name="subscribe" type="Boolean" mandatory="false">
```

Comment:
remove `subscribe` parameter, since Core does nothing with this parameter if sent from the HMI